.github/workflows/test-docs.yml

```
@@ -0,0 +1,33 @@
+name: test-docs
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - "main"
+jobs:
+  test-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)